### ISSUE TITLE
Support syntax or config for customizing the Avro schema name for schemas generated by KSQL

### ISSUE THREAD

**Issue by rodesai** [confluentinc/ksql#1431]
Currently, KSQL assigns a default name to the avro schemas it generates. We should provide a way for the user to specify the name. This would let them use the Specific deserializer with a deserialized class name of their choice.

**Comment by apurvam** [comment]
This should be fixed by #1863, right @rodesai . Can we close this out if so?

**Comment by rodesai** [comment]
Yep, closing
